<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Converter</class>
 <widget class="QMainWindow" name="Tela_Converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botao_voltar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>VOLTAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/voltar.png</normaloff>../images/voltar.png</iconset>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>821</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GENBANK - FASTA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FASTQ - FASTA</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TIPO DE CONVERS√ÉO: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_abrir_arquivo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>118</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ABRIR ARQUIVO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_converter">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CONVERTER</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="setar_sequencia">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>110</y>
      <width>781</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
